system/views/formcontrols/selectDerivatives/index.cfm

```diff
@@ -0,0 +1,50 @@
+<cfscript>
```

Comment:
I don't think we need this file. After building the arrays of `values` and `labels` in the handler, you should be able to just render the "select" form control. Use extra classes for any javascript that needs to interact with the field. Hardcoding the ID of the field means there can be only one picker per page which should be avoided as an approach.